app/models/ability.rb

```diff
@@ -0,0 +1,17 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    return unless user.present?
```

Comment:
- [ ] I would suggest you update this line to `user ||= User.new # guest user (not logged in)` which means that a guest user can at least still see/read posts

Reply:
Hi @GzimAsani, thank you for taking the time to review our work. Do you mean the three lines are the ones to be replaced?

Reply:
hi @alphayowakarindi, no I meant only this line
 `return unless user.present?`

Reply:
@GzimAsani Okay, thanks. Kindly respond to the other review you have recommended.